doc/source/getting_started/docker/index.rst

```diff
@@ -0,0 +1,61 @@
+.. _ref_docker:
+
+Geometry service using Docker
+=============================
+
+What is Docker?
+---------------
+
+Docker is an open platform for developing, shipping, and running apps in a
+containerized way.
+
+Containers are standard units of software that package up code and all its dependencies
+so the app runs quickly and reliably from one computing environment to another.
```

Comment:
```suggestion
Containers are standard units of software that package the code and all its dependencies
so that the app runs quickly and reliably from one computing environment to another.
```